buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/measurement/MetricsReportUploader.groovy

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.gradle.plugins.measurement
+
+import org.gradle.api.Project
+
+/** A helper class for uploading a metric report to the metrics service. */
+class MetricsReportUploader {
+    private static final def METRICS_SERVICE_URL = System.getenv("METRICS_SERVICE_URL")
+
+    /** Uploads the given metric report .*/
+    static void upload(Project project, String report) {
+        if (!System.getenv().containsKey("FIREBASE_CI")) {
+            project.logger.quiet "Metrics upload is enabled only on CI."
+            return
+        }
+
+        def owner = null
+        def repo = null
+        def baseCommit = null
+        def headCommit = null
+        def pullRequest = null
+
+        if (System.getenv().containsKey("PROW_JOB_ID")) {
```

Comment:
Reverse the logic to reduce nesting
```
if (!system.getenv...) {
project.logger.quiet
return ?
} 
...
```
the continue with the rest of the code

Reply:
Done.